.github/workflows/ci.yml

```
@@ -35,16 +35,25 @@ jobs:
         DISPLAY: ':99.0'
       timeout-minutes: 10
       
     - name: Run tests (Windows/macOS)
       if: runner.os != 'Linux'
-      run: npm test
+      run: npm run test:coverage
       timeout-minutes: 10
       
+    - name: Upload coverage to Codecov
+      if: matrix.os == 'ubuntu-latest' && matrix.node-version == '18.x'
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)